### PR TITLE
MBS-8005 fix documentation

### PR DESCRIPTION
**Pull request by int02h**

### CODE REVIEW

File: docs/content/docs/ide/Benchmarking.md [hunk 1]
```diff
@@ -10,15 +10,13 @@ type: docs
...
-переводит строку и не выбирать опции в выпадающем меню автокомплита.
-
-К самому плагину оставлен
-[комментарий](https://intellij-support.jetbrains.com/hc/en-us/articles/207241225/comments/360001111259),
-а также заведен [issue](https://intellij-support.jetbrains.com/hc/en-us/requests/2577588) в трекер
-IntelliJ.
+Чтобы плагин работал корректно на macOS, необходимо разрешить Android Studio/IntelliJ IDEA доступ к функциям
+Accessibility системы. Для этого:
+
+1. Открываем *System Preferences* -> *Security & Privacy*
```
Comment (eugene-krivobokov) [inline docs/content/docs/ide/Benchmarking.md:16]:
```suggestion
1. Открываем **System Preferences > Security & Privacy**
```

Use this format for consistency: **Menu > Item**

https://avito-tech.github.io/avito-android/docs/contributing/docs/#interaction-with-ui

File: docs/content/docs/ide/Benchmarking.md [hunk 2]
```diff
@@ -10,15 +10,13 @@ type: docs
...
+1. Выбираем вкладку *Privacy*
```
Comment (eugene-krivobokov) [inline docs/content/docs/ide/Benchmarking.md:17]:
```suggestion
1. Выбираем вкладку **Privacy**
```

File: docs/content/docs/ide/Benchmarking.md [hunk 3]
```diff
@@ -10,15 +10,13 @@ type: docs
...
+1. В списке слева находим пункт *Accessibility*
```
Comment (eugene-krivobokov) [inline docs/content/docs/ide/Benchmarking.md:18]:
```suggestion
1. В списке слева находим пункт **Accessibility**
```

**Review by dsvoronin — APPROVED** [review]

**Review by eugene-krivobokov — CHANGES_REQUESTED** [review]

File: docs/content/docs/ide/Benchmarking.md (hunk 1 above)
Comment (int02h) [reply docs/content/docs/ide/Benchmarking.md:16]: Thanks, I will fix all cases.

**Review by eugene-krivobokov — APPROVED** [review]

**Pull request merged**